src/App.tsx

```diff
@@ -1,11 +1,72 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import * as todosApi from './api/todos';
+import { Todo } from './types/Todo';
+import cn from 'classnames';
+import { Status } from './types/Status';
+import { getFilterTodos } from './utils/getFilterTodos';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { wait } from './utils/fetchClient';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [titles, setTitles] = useState('');
+  const [errorMessage, setErrorMessage] =
+    useState<string>('');
+  const [filter, setFilter] = useState<Status>(Status.All);
+
+  useEffect(() => {
+    todosApi
+      .getTodos()
+      .then(setTodos)
+      .catch(() => {
+        setErrorMessage(`Unable to load todos`);
+        wait(3000).then(() => {
+          setErrorMessage('');
+        });
+      });
+  }, []);
+
+  function addTodo({ title, completed, userId }: Todo) {
+    todosApi
+      .createTodos({ title, completed, userId })
+      .then(newTodo => {
+        setTodos(currentTodos => {
+          return [...currentTodos, newTodo];
+        });
+      });
+  }
```

Comment:
You don't need to do this in this task
But if you did it
What about `catch`?

Reply:
> You don't need to do this in this task But if you did it What about `catch`?

Not fixed
Also, use the same style for your functions, because now I see some arrow functions, but this one is function declaration